webapp/src/root_fetcher.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {useEffect} from 'react';
+import {useSelector, useDispatch} from 'react-redux';
+
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/common';
+
+import {fetchNote} from 'client';
+import {setNoteValue} from 'redux_connectors';
+
+const RootFetcher = () => {
+    const dispatch = useDispatch();
+    const currentChannelId = useSelector(getCurrentChannelId);
+
+    useEffect(() => {
+        const fetch = async () => {
+            const fetchedNote = await fetchNote(currentChannelId);
+            dispatch(setNoteValue(currentChannelId, fetchedNote));
+        };
+
+        fetch();
+    }, [currentChannelId]);
```

Comment:
0/5 Should we pass `dispatch` here as well?

Reply:
I think there's no need to do it, right? That warning some checkers give seems off to me.